.changeset/thick-ads-smash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': minor
```

Comment:
We use `minor` for breaking changes and `patch` for everything else while we're on `0.x`. I'd argue this isn't a breaking change but not certain about that.

Reply:
Great! Thanks! I wasn't aware about the deviation from semver. Will update.

Reply:
It's arguably part of the `semver` spec when we're `<1.0.0` https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase

Reply:
Right, of course! Good point.